hw2/05-form.html

```diff
@@ -10,11 +10,54 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
+  <body class="bg-dark">
     <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
-      </form>
+      
+<div class = "container my-5">
+<form class="card bg-light mb-5 mx-auto py-3 px-3"
+style="max-width: 30rem; height: 38rem">
+  <h2 class="text-center"><strong>Exercise 05 - Form</strong></h1>
+  <div class="mb-3">
+    <label for="inputName" class="form-label"><strong>Full Name:</strong></label>
+    <input type="name" class="form-control" id="InputName">
+  </div>
+  <div class="mb-3">
+    <label for="InputEmail" class="form-label"><strong>Email:</strong></label>
+    <input type="email" class="form-control" id="InputEmail">
+  </div>
+  <div class="mb-3">
+    <label for="InputRegistration" class="form-label"><strong>Registration Status:</strong></label>
+    <select id="Registration" class="form-select">
+      <option>Choose an Option</option>
```

Comment:
I believe for the options, we should include (Registered, Waitlist, Other) as the professor mentioned this during her lecture.

Reply:
I didn't catch this during lecture, It wasn't specifically stated in the assignment so I did not know to add it. Thanks